app/models/conversation.rb

```diff
@@ -0,0 +1,42 @@
+class Conversation < ApplicationRecord
+  before_validation :assign_short_id, :on => :create
+
+  belongs_to(
```

Comment:
Style nit: check out the top of `app/models/user.rb` to see how these are formatted.